packages/DraftEditor/src/DraftPlugins/index.tsx

```diff
@@ -0,0 +1,114 @@
+import createAlignmentPlugin from "draft-js-alignment-plugin";
+import "draft-js-alignment-plugin/lib/plugin.css";
+import {
+  BlockquoteButton,
+  BoldButton,
+  CodeBlockButton,
+  CodeButton,
+  HeadlineOneButton,
+  HeadlineThreeButton,
+  HeadlineTwoButton,
+  ItalicButton,
+  OrderedListButton,
+  UnderlineButton,
+  UnorderedListButton
+} from "draft-js-buttons";
+import createBlockDndPlugin from "draft-js-drag-n-drop-plugin";
+import createFocusPlugin from "draft-js-focus-plugin";
+import "draft-js-focus-plugin/lib/plugin.css";
+import createImagePlugin from "draft-js-image-plugin";
+import "draft-js-image-plugin/lib/plugin.css";
+import createInlineToolbarPlugin from "draft-js-inline-toolbar-plugin";
+import "draft-js-inline-toolbar-plugin/lib/plugin.css";
+import createLinkifyPlugin from "draft-js-linkify-plugin";
+import "draft-js-linkify-plugin/lib/plugin.css";
+import { composeDecorators } from "draft-js-plugins-editor";
+import createResizeablePlugin from "draft-js-resizeable-plugin";
+import React, { useEffect } from "react";
+
+const focusPlugin = createFocusPlugin();
+const blockDndPlugin = createBlockDndPlugin();
+const linkifyPlugin = createLinkifyPlugin();
+const resizeablePlugin = createResizeablePlugin();
+const alignmentPlugin = createAlignmentPlugin();
+const inlineToolbarPlugin = createInlineToolbarPlugin();
+
+const { InlineToolbar } = inlineToolbarPlugin;
+const { AlignmentTool } = alignmentPlugin;
+
+const imagePlugin = createImagePlugin({
+  decorator: composeDecorators(
+    blockDndPlugin.decorator,
+    alignmentPlugin.decorator,
+    resizeablePlugin.decorator,
+    focusPlugin.decorator
+  )
+});
+
+const plugins = [
+  focusPlugin,
+  inlineToolbarPlugin,
+  blockDndPlugin,
+  imagePlugin,
+  linkifyPlugin,
+  alignmentPlugin,
+  resizeablePlugin
+];
+
+const HeadlinesPicker = (props: any) => {
+  useEffect(() => {
+    setTimeout(() => {
+      window.addEventListener("click", onWindowClick);
```

Comment:
this event is added but is never remove, that can lead to some serious performance issues if its not handled and removed when it is not needed anymore